app/assets/stylesheets/custom.css.scss

```diff
@@ -44,6 +44,45 @@ body {
   }
 }
 
+.smfont {
+  font-size: 0.9em;
+}
+.warning {
+  color: red;

```

Comment:
Color `red` should be written in hexadecimal form as `#ff0000`<br>Color literals like `red` should only be used in variable declarations; they should be referred to via variable everywhere else.